server/api/initial_data.py

```diff
@@ -867,6 +871,30 @@ def _migrate_model_monitoring_jobs(db, db_session):
     )
 
 
+def _perform_version_7_data_migrations(
+    db: server.api.db.sqldb.db.SQLDB, db_session: sqlalchemy.orm.Session
+):
+    _migrate_project_summaries(db, db_session)
+
+
+def _migrate_project_summaries(db, db_session):
+    # Here we want to iterate other all projects and create the related project summary.
+    # We do this because we create ProjectSummary record only when creating project.
+    # Because we upgrade from a version that doesn't have ProjectSummary, we need to create them manually now.
+    projects = db.list_projects(
+        db_session, format_=mlrun.common.formatters.ProjectFormat.name_only
+    )
+    for project_name in projects.projects:
+        summary = mlrun.common.schemas.ProjectSummary(
+            name=project_name,
+        )
+        project_summary = ProjectSummary(
+            project=project_name,
+            summary=summary.dict(),
+        )
+        db._upsert(db_session, [project_summary], ignore=True)
```

Comment:
```suggestion
    project_summaries = [
        ProjectSummary(
            project=project_name,
            summary=mlrun.common.schemas.ProjectSummary(name=project_name).dict()
        )
        for project_name in projects.projects
    ]
    
    db._upsert(db_session, project_summaries, ignore=True)
```

This way, you commit all changes together.